### ISSUE TITLE
[TASK]: Add Cache to "Test" job in Workflow

### ISSUE THREAD

**Issue by sudo-adduser-jordan** [webdevcody/code-racer#560]
### Task Summary

The "Test" job in the workflow should also be cached. This will save about 30 secs of time per execution.

**Comment by TyrinH** [comment]
@sudo-adduser-jordan are you working on this? If not I'd like to since this is all new to me in the codebase

**Comment by sudo-adduser-jordan** [comment]
No, I am not. I'm not aware of anyone else either. Feel free to work on it. @TyrinH

**Comment by sudo-adduser-jordan** [comment]
This may be helpful. 

https://github.com/nektos/act
